lib/1.4/dml-builtins.dml

```diff
@@ -2785,6 +2785,12 @@ template field is (object, _bitsize, _lsb, init_val, shown_desc, _get, _set) {
         set_val(init_val);
     }
 
+    shared method _full_name() -> (const char *) {
+        local uint64 regname = strlen(reg._qname());
+        local const char *qname = _qname();
+        return qname + regname + 1;
```

Comment:
Just want to note that it's a good thing that this is 1.4 exclusive, otherwise this would be a confidentiality leak. (Hell, even instantiating the template would be). A 1.2 model importing a 1.4 module is not a concern as then the dml-builtins.dml in use will be that of DML 1.2.